python/cogs/spam_blocker.py

```diff
@@ -446,6 +446,38 @@ async def remove_spammer_item(self, ctx, _id:int):
             await ctx.send(embed=embed)
 
 
+    @spammer.command(
+        name='search',
+        aliases=['user', 'history']
+    )
+    async def spammer_search(self, ctx, member: Member = None):
+        """Search member name to see if they have been caught phishing before."""
+        member_id = member.id
+        async with async_session() as db:
+            async with db.begin():
+                scd = SpammerDAL(db)
+                res = await scd.search_spammer(member_id)
+                if not res:
+                    embed = Embed(
+                        color=0x13DC51,
+                        title='Criminal History Check',
+                        description=f'```{await self.client.fetch_user(member_id)} is clean.```'
+                    )
+                    return await ctx.send(embed=embed)
+
+        rules_broken = '\n'.join([
+            f'{row.id} | ' +
+            f'{await self.client.fetch_user(row.member)} | ' +
+            f'{row.regex}' for row in res
+        ])
+        embed = Embed(
```

Comment:
I would consider creating the embed once, and updating the description based  on `res` to reduce duplication.